### PR TITLE
update/add descriptions

### PR DESCRIPTION
**Pull request by mpeck**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ permissions:
 commands:
   alert:
     executable: /home/bundle/cog-command
+    description: "Triggers alerts"
```
Comment (vanstee) [inline config.yaml]: Not a big deal, but might be nice to stick to the same verb style for all of these `Triggers alerts` vs `Trigger alerts` and `Acknowledge incidents` vs `Acknowledges incidents`.

File: config.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ commands:
         short_flag: s
   oncall:
     executable: /home/bundle/cog-command
-    description: "Returns the current oncall for all, or a specified service"
+    description: Returns the current oncall for all, or a specified service
```
Comment (vanstee) [inline config.yaml]: Maybe `s/Returns/Retrieve/` to stay consistent. Totally not a big deal, just feeling nitpicky.

**Comment by vanstee** [comment]
:+1:

File: config.yaml [hunk 3]
```diff
@@ -40,9 +41,10 @@ commands:
         short_flag: s
   oncall:
     executable: /home/bundle/cog-command
+    description: Retrieve the current oncall for all, or a specified service
```
Comment (christophermaier) [inline config.yaml]: Rewording suggestion: "Find out who is on-call for all services, or for a specific service"

**Pull request merged**